packages/swarm-components/lib/Select.js

```diff
@@ -11,22 +11,29 @@ const Select = props => {
     id,
     label,
     name,
-    required,
+    requiredText,
```

Comment:
Hmmm, this file for whatever reason is being updated. It might need to be `git rm` since it was probably accidentally checked in in a prior ticket? cc @hsbacot